.github/workflows/test.yaml

```diff
@@ -41,13 +41,20 @@ jobs:
         if: matrix.operating-system == 'ubuntu-latest'
 
       - name: Lint
+        id: lint
         uses: golangci/golangci-lint-action@v3.7.0
         with:
           version: v1.54
-          args: --deadline=30m
+          args: --deadline=30m --out-format=line-number
           skip-cache: true # https://github.com/golangci/golangci-lint-action/issues/244#issuecomment-1052197778
         if: matrix.operating-system == 'ubuntu-latest'
 
+      - name: Check if linter failed
+        run: |
+          echo "Linter failed, please run mage lint:fix to correct any errors"
```

Comment:
nit

```suggestion
          echo "Linter failed, please run 'mage lint:fix' to correct any errors"
```

Reply:
nit: Can we change the wording to say that some errors cannot be fixed by "mage lint:fix" and that this command may help? Otherwise, people misunderstand `mage lint:fix` should fix any errors, and the test can fail again.

Reply:
Updated https://github.com/aquasecurity/trivy/pull/5247/commits/515e88f39cb9f1fb96881863ff4455dd648a59c4 let me know if it's good.

Reply:
Thanks!